.github/workflows/deployment.yml

```diff
@@ -2,50 +2,117 @@ name: Deployment
 
 on:
   push:
-    tags:
-      - v5r0-pre[0-9]+
   pull_request:
+  workflow_dispatch:
+    inputs:
+      check-ci:
+        description: "Require the CI to have passed for this commit"
+        required: true
+        default: "yes"
+      version:
+        description: "Override the release version number (e.g. 5.0.0a5)"
 
 jobs:
   deploy-pypi:
     name: PyPI deployment
     runs-on: "ubuntu-latest"
     if: github.event_name != 'push' || github.repository == 'DIRACGrid/WebAppDIRAC'
+    defaults:
+      run:
+        shell: bash -l {0}
     steps:
       - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.PAT || github.token }}
```

Comment:
IIUC `PAT` more limited than standard `github.token`? I'm just trying to understand why just not use the standard one.

Reply:
`secrets.PAT` is less limited and able to push to protected braches

Reply:
However it's not defined for PR builds so we fall back to the standard one when just testing the release building machinery.